<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiasporaClusterDialog</class>
 <widget class="QWidget" name="DiasporaClusterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0,0,10">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string># of Solar Systems</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinNumSolSys">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>    Cluster name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lblClusterName">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGraphicsView" name="clusterView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>227</red>
              <green>227</green>
              <blue>227</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QToolButton" name="btnZoomIn">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/new/Bitmaps/pics/zoomin.png</normaloff>:/new/Bitmaps/pics/zoomin.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderZoom">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>250</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnZoomOut">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/new/Bitmaps/pics/zoomout.png</normaloff>:/new/Bitmaps/pics/zoomout.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QToolButton" name="btnRotateLeft">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/new/Bitmaps/pics/rotateleft.png</normaloff>:/new/Bitmaps/pics/rotateleft.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderRotate">
           <property name="minimum">
            <number>-350</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRotateRight">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/new/Bitmaps/pics/rotateright.png</normaloff>:/new/Bitmaps/pics/rotateright.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="ClusterItemEditor" name="cliEditor" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClusterItemEditor</class>
   <extends>QWidget</extends>
   <header location="global">clusteritemeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>lstCluster_selected(QModelIndex)</slot>
 </slots>
</ui>
